planet.rb

```diff
@@ -0,0 +1,19 @@
+class Planet
+    attr_reader :name, :color, :mass_kg, :distance_from_sun_km, :fun_fact
+
+    def initalize(name, color, mass_kg, distance_from_sun_km, fun_fact)
```

Comment:
Initialize is misspelled

```suggestion
    def initialize(name, color, mass_kg, distance_from_sun_km, fun_fact)
```